legend-engine-xts-analytics/legend-engine-xts-analytics-mapping/legend-engine-xt-analytics-mapping-pure/src/main/resources/core_analytics_mapping/modelCoverage/mappedEntityBuilder.pure

```diff
@@ -281,18 +290,47 @@ function meta::analytics::mapping::modelCoverage::buildEntity(
          ); //TODO check how types work with qp's.we n eed to filter on whether the type is root supported currently
    );
 
-   let autoMappedProperties = if($setImplementation->instanceOf(PureInstanceSetImplementation),
+   // handle auto mapped properties 
```

Comment:
this naturally feels wrong. I would expect the binding to be exposed in the classMappings as every other target type is. This way it would be a very standard class mapping handler on the SemiStructuredEmbeddedRelationalInstanceSetImplementation and these functions keep their initial contract

Reply:
Hello John, thanks for reviewing this PR.  SemiStructuredEmbeddedRelationalInstanceSetImplementation needs to be handled as an auto-mapped mapping due to this newly added commit
https://github.com/finos/legend-engine/pull/2141